language/de/email/forum_notify.txt

```diff
@@ -1,11 +1,9 @@
-Subject: Benachrichtigung über neue Antwort — „{FORUM_NAME}“
+Subject: Neue Antwort in „{FORUM_NAME}“
```

Comment:
Neu ist die Antwort ja definitiv, sonst gäbe es keine E-Mail. Brauchen wir das "neue" daher unbedingt?

Reply:
Einheitlich sollten wir es auf jeden Fall machen :-)
Meiner Ansicht kann das neue weg - auch wenn es natürlich alte Antworten gibt...

Reply:
Dann mach es weg. 👍